src/js/themeOverride.js

```diff
@@ -266,39 +283,88 @@ export const themeOverride = {
     },
     icons: {
       include: {
-        icon: Success,
+        icon: TickCircle,
         extend: {
-          color: 'status-ok',
+          color: 'accent-1',
           size: 'large',
         },
       },
       exclude: {
-        icon: Block,
+        icon: TickCircle,
         extend: {
-          color: 'status-error',
+          color: 'error',
           size: 'large',
         },
       },
     },
   },
+  pagination: {
```

Comment:
ToDo: To be moved to main theme

Reply:
Moved Upload theme to main theme,
Pagination and other themes are overridden in this file